lib/icalendar/parser.rb

```diff
@@ -119,14 +132,16 @@ def component_class
       @component_class ||= Icalendar::Calendar
     end
 
+    PARSE_COMPONENT_KLASS_NAME_GSUB_REGEX = /\AV/
+
     def parse_component(component)
       while (fields = next_fields)
         if fields[:name] == 'end'
-          klass_name = fields[:value].gsub(/\AV/, '').downcase.capitalize
+          klass_name = fields[:value].gsub(PARSE_COMPONENT_KLASS_NAME_GSUB_REGEX.freeze, '').downcase.capitalize
```

Comment:
Is there a reason we'd want to put `.freeze` here, rather than on line 135?

Reply:
I can't think of a reason; I think it can be moved to line 135; will do!